packages/compass-aggregations/src/components/pipeline-explain/explain-results.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { css, spacing, Card } from '@mongodb-js/compass-components';
+import { Document } from '@mongodb-js/compass-crud';
+import HadronDocument from 'hadron-document';
+
+import type { ExplainData } from '../../modules/explain';
+import { ExplainQueryPerformance } from './explain-query-performance';
+
+type ExplainResultsProps = {
+  plan: ExplainData['plan'];
+  stats?: ExplainData['stats'];
+};
+
+const containerStyles = css({
+  display: 'flex',
+  flexDirection: 'column',
+  gap: spacing[4],
+});
+
+const cardStyles = css({
+  // 170px works with minimum-height of compass
+  // todo: handle height for bigger sized compass
```

Comment:
As an addition to this TODO, currently because "copy to clipboard" button is part of the Document component and not the card, the whole thing scrolls including the button:

<img width="576" alt="image" src="https://user-images.githubusercontent.com/5036933/169251144-33fc4292-2ce3-46bd-9846-2898533c2f3c.png">

I think we might want to set the height here on the document fields container and not the wrapper card as a way to prevent it, but there is no way to do it now and it's a very small thing so totally can be addressed later

Reply:
sure :)